### PR TITLE
chore(doc): latest and stable cleanup

### PR DESCRIPTION
**Pull request by gjanco**
rewording k8s related references to stable as we have removed the tag.

LOG-13374

### CODE REVIEW

File: docs/KUBERNETES.md [hunk 1]
```diff
@@ -136,27 +136,25 @@ Older versions of our configurations do not provide these labels. In that case,
...
 ```
 
-* `latest` - Update with each new revision including public betas.
-* `stable` - Updates with each major, minor, and patch version updates.
-* `2` - Updates with each minor and patch version updates under `2.x.x`.
-* `2.2` - Updates with each patch version update under `2.2.x`.
-* `2.2.0` - Targets a specific version of the agent.
+* `3` - Updates with each minor and patch version updates under `3.x.x`.
+* `3.5` - Updates with each patch version update under `3.5.x`.
+* `3.5.1` - Targets a specific version of the agent.
```
Comment (dkhokhlov) [inline docs/KUBERNETES.md:157]: the latest release is 3.6.0?

File: docs/KUBERNETES.md [hunk 2]
```diff
@@ -136,27 +136,25 @@ Older versions of our configurations do not provide these labels. In that case,
...
+The image contains the actual agent code that is run on the Pods created by the DaemonSet. New versions of the agent always strive for backwards compatibility with old configuration versions. Any breaking changes will be outlined in the [change log](https://docs.mezmo.com/changelog). We always recommend upgrading to the latest configuration to guarantee access to new features.
 
 The upgrade path for the image depends on which image tag you are using in your DaemonSet.
 
-If your DaemonSet is configured with `logdna/logdna-agent:stable`, the default configuration setting, then restarting your Pods will trigger them to pull down the latest stable version of the LogDNA agent image.
+If your DaemonSet is configured with `logdna/logdna-agent:3`, or some other major version number, then restarting your Pods will trigger them to pull down the latest minor version of the LogDNA agent image for this major version (in this example `3`).
```
Comment (stsantilena) [inline docs/KUBERNETES.md:143]: Nit: how about "...pull down the latest minor version for this major version (in this example `3`) of the LogDNA agent image."

File: docs/OPENSHIFT.md [hunk 3]
```diff
@@ -92,28 +92,26 @@ Older versions of our configurations do not provide these labels. In that case,
...
+The image contains the actual agent code that is run on the Pods created by the DaemonSet. New versions of the agent always strive for backwards compatibility with old configuration versions. Any breaking changes will be outlined in the [change log](https://docs.mezmo.com/changelog). We always recommend upgrading to the latest configuration to guarantee access to new features.
 
 The upgrade path for the image depends on which image tag you are using in your DaemonSet.
 
-If your DaemonSet is configured with `logdna/logdna-agent:stable`, the default configuration setting, then restarting your Pods will trigger them to pull down the latest stable version of the LogDNA agent image.
+If your DaemonSet is configured with `logdna/logdna-agent:3`, or some other major version number, then restarting your Pods will trigger them to pull down the latest minor version of the LogDNA agent image for this major version (in this example `3`).
```
Comment (stsantilena) [inline docs/OPENSHIFT.md:99]: Same suggestion as above.

**Review by stsantilena — APPROVED** [review]
Just one minor nit. LGTM!

File: docs/KUBERNETES.md (hunk 1 above)
Comment (gjanco) [reply docs/KUBERNETES.md:157]: This is just an example...so we will leave this even as we update.  I just wanted to have a more relevant example.

**Pull request closed without merging**